facebook-common/src/main/java/com/facebook/login/WebViewLoginMethodHandler.java

```diff
@@ -30,6 +30,7 @@
 import com.facebook.FacebookException;
 import com.facebook.internal.FacebookDialogFragment;
 import com.facebook.internal.ServerProtocol;
+import com.facebook.internal.Utility;
```

Comment:
Does this class use the Utility?

Reply:
Not anymore, good catch.

Reply:
I removed this line but now github is showing 3 extra commits; I spent 30m or so trying to merge / squash them in just 1, but I gave up :-( Let me know if this is ok, or how I should proceed (worst case scenario I could create a new pull request, but then we would lose the conversation history.

BTW, you can verify the changes by setting installing the Autofill Framework sample (https://github.com/googlesamples/android-AutofillFramework), setting its "Multi-Datasset Autofill Service" as the user's autofill service  (using Settings -> System -> Language & Input -> Advanced -> Autofill Service), then launch the login sample, select "Log in with Facebook", then "Continue with Facebook", then tap on "Email or Phone" - you should get a toast saying the service could not verifiy the domain, and 'adb logcat AutofillSample:* *:s" should also have some output (OTOH, without the patch, the service would not be triggered).

Reply:
Yeah, why don't we re-create the PR on top of present day dev branch. Im not quite sure what is going on with all the other commits but just to be safe let's get them out of there. You don't have to worry about losing the conversation here. I will keep track of it in this closed PR. Thanks. 

Thanks for the info on testing. That is going to be SUPER helpful today when I run it through the ringers.

Reply:
OK, filed https://github.com/facebook/facebook-android-sdk/pull/518 , looks good now (just 4 files change, no previous commit included).

I will try to fork the Autofill service to work with it - right now it doesn't because:

1.The Digital Assets Link verification fail because the app is not registered on Facebook's site.
2.It doesn't find any autofillable fields

These 2 issues are expected from the reference service implementation; a "real" service would be more sophisticated to handle them. I have a local service that works, but it's not based in the sample service...